backend/src/db/models/message.ts

```diff
@@ -0,0 +1,32 @@
+import { DataTypes, Sequelize } from "sequelize";
+import { sequelize } from "../../config/database";
+
+const Message = sequelize.define("message", {
+  id: {
+    type: DataTypes.UUID,
+    allowNull: false,
+    unique: true,
+    primaryKey: true,
+  },
+  message: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  created_at: {
+    type: DataTypes.DATE,
+      allowNull: false,
+    defaultValue: Date.now(),
+  },
+  updated_at: {
+    type: DataTypes.DATE,
+      allowNull: false,
```

Comment:
Format the code.

Turn on format on save in vs code. I think there is an issue with autosave and autoformat working together.

To catch these issues, we have to add [a linter](https://www.sonarsource.com/learn/linter/#:~:text=A%20linter%20is%20a%20developer,issues%20to%20improve%20code%20quality.) at last. Please keep a note of this and plz remind me in I forget..